Makefile

```diff
@@ -16,7 +16,31 @@ GROUP_NAME ?= $(shell id -g -n)
 
 COVERAGE ?= $(REPO_ROOT_DIR)/coverage
 
-VERBOSITY ?= 6
+VERBOSITY ?= 0
```

Comment:
Let me know if you'd like me to change this back. Running it locally is frustrating with it at 6